src/pkg/dir.targets

```diff
@@ -1,7 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" InitialTargets="CheckForBuildTools" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <Import Project="..\dir.targets" />
+  <Import Project="..\dir.targets" />
+  <PropertyGroup>
+    <RuntimeIdGraphDefinitionFile>$(PackagesDir)$(PlatformPackageId.ToLowerInvariant())\$(MicrosoftNETCorePlatformsPackageVersion)\runtime.json</RuntimeIdGraphDefinitionFile>
+  </PropertyGroup>
+
+  <!-- Create deb tool package and set up the project that consumes it as a CLI tool. -->
+  <Target Name="InitializeDotnetDebTool">
+    <!--
+      Build the deb tool and initialize the consumer project. Using an MSBuild task helps this
+      to only happen once if parallel projects depend on this target.
+    -->
+    <MSBuild Projects="$(MSBuildThisFileDirectory)deps\init-deb-tool-consumer.proj" />
+  </Target>
+
+  <!-- Check if the deb package build tool prereq is available. -->
+  <Target Name="TestDebuild">
+    <!-- run Debuild -->
+    <Exec Command="/usr/bin/env debuild -h" ContinueOnError="true">
+      <Output TaskParameter="ExitCode" PropertyName="DebuildExitCode" />
+    </Exec>
+
+    <!-- Check if it returned 0 -->
     <PropertyGroup>
-        <RuntimeIdGraphDefinitionFile>$(PackagesDir)$(PlatformPackageId.ToLowerInvariant())\$(MicrosoftNETCorePlatformsPackageVersion)\runtime.json</RuntimeIdGraphDefinitionFile>
+      <DebuildPresent>false</DebuildPresent>
+      <DebuildPresent Condition=" '$(DebuildExitCode)' == '0' ">true</DebuildPresent>
     </PropertyGroup>
+
+    <Message Condition=" '$(DebuildPresent)'  != 'true' "
+             Text="Debuild Not found, Debian packages will not be built."
+             Importance="High" />
+  </Target>
+
+  <!-- Check if the RPM package build tool prereq is available. -->
+  <Target Name="TestFPMTool">
+    <!-- run FPM  -->
+    <Exec Command="/usr/bin/env fpm -h" ContinueOnError="true">
+      <Output TaskParameter="ExitCode" PropertyName="FPMExitCode" />
+    </Exec>
+
+    <!-- Check if it returned 0 -->
+    <PropertyGroup>
+      <FPMPresent>false</FPMPresent>
+      <FPMPresent Condition=" '$(FPMExitCode)' == '0' ">true</FPMPresent>
+    </PropertyGroup>
+
+    <!-- Workaround for Jenkins machines that don't have the necessary packages https://github.com/dotnet/core-setup/issues/2260 -->
```

Comment:
Ah, didn't get rid of the comment when I moved this one. Yes, we're off Jenkins, but it's not specific to Jenkins, it's just about what the machine/image has available. I removed the comment from the other target because it's just a capability check, not really a workaround IMO.

But... it does makes me uneasy to skip RPM/DEB package builds based on this. For example, in our official build, if we switched to a broken RPM build image (`fpm` not in PATH or something), I think we would silently stop producing RPMs. I'm not going to tackle this in this PR though, filing an issue.